features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/recorder/mapper/ImageViewMapper.kt

```diff
@@ -13,26 +13,48 @@ import com.datadog.android.sessionreplay.internal.recorder.densityNormalized
 import com.datadog.android.sessionreplay.internal.utils.ImageViewUtils
 import com.datadog.android.sessionreplay.model.MobileSegment
 import com.datadog.android.sessionreplay.recorder.MappingContext
-import com.datadog.android.sessionreplay.recorder.mapper.BaseAsyncBackgroundWireframeMapper
 import com.datadog.android.sessionreplay.utils.AsyncJobStatusCallback
 import com.datadog.android.sessionreplay.utils.ColorStringFormatter
 import com.datadog.android.sessionreplay.utils.DrawableToColorMapper
 import com.datadog.android.sessionreplay.utils.ImageWireframeHelper
 import com.datadog.android.sessionreplay.utils.ViewBoundsResolver
 import com.datadog.android.sessionreplay.utils.ViewIdentifierResolver
 
-internal class ImageViewMapper(
-    private val imageViewUtils: ImageViewUtils,
-    viewIdentifierResolver: ViewIdentifierResolver,
-    colorStringFormatter: ColorStringFormatter,
-    viewBoundsResolver: ViewBoundsResolver,
-    drawableToColorMapper: DrawableToColorMapper
-) : BaseAsyncBackgroundWireframeMapper<ImageView>(
-    viewIdentifierResolver,
-    colorStringFormatter,
-    viewBoundsResolver,
-    drawableToColorMapper
-) {
+/**
+ * A [WireframeMapper] implementation to map an [ImageView] component.
+ */
+open class ImageViewMapper : BaseAsyncBackgroundWireframeMapper<ImageView> {
+    private val imageViewUtils: ImageViewUtils
+
+    @Suppress("Unused") // used by external mappers
+    constructor(
+        viewIdentifierResolver: ViewIdentifierResolver,
+        colorStringFormatter: ColorStringFormatter,
+        viewBoundsResolver: ViewBoundsResolver,
+        drawableToColorMapper: DrawableToColorMapper
+    ) : this(
+        viewIdentifierResolver,
+        colorStringFormatter,
+        viewBoundsResolver,
+        drawableToColorMapper,
+        ImageViewUtils
+    )
+
+    internal constructor(
+        viewIdentifierResolver: ViewIdentifierResolver,
+        colorStringFormatter: ColorStringFormatter,
+        viewBoundsResolver: ViewBoundsResolver,
+        drawableToColorMapper: DrawableToColorMapper,
+        imageViewUtils: ImageViewUtils
+    ) : super(
+        viewIdentifierResolver,
+        colorStringFormatter,
+        viewBoundsResolver,
+        drawableToColorMapper
+    ) {
+        this.imageViewUtils = imageViewUtils
+    }
```

Comment:
> [!NOTE]
> This could be rewritten using a default value parameter: 

```kotlin
open class ImageViewMapper (
    viewIdentifierResolver: ViewIdentifierResolver,
    colorStringFormatter: ColorStringFormatter,
    viewBoundsResolver: ViewBoundsResolver,
    drawableToColorMapper: DrawableToColorMapper,
    private val imageViewUtils: ImageViewUtils = ImageViewUtils
) : BaseAsyncBackgroundWireframeMapper<ImageView> {
 
```